mars/deploy/oscar/tests/test_ray.py

```diff
@@ -578,7 +578,7 @@ async def remote(self):
     class FakeActor:
         state = FakeActorMethod()
 
-    def _get_actor(*args):
+    def _get_actor(*args, **kwargs):
```

Comment:
A little confused. It doesn't seem necessary to do this change.

Reply:
In some version of ray, ray will pass `namespace` named parameter to this method. Without it, the method call will fail